### DISCUSSION TITLE
Call for Entries: Présentez vos projets, produits ou initiatives

### DISCUSSION THREAD

**Discussion by christianheimann** [e-id-admin/governance-sounding-board#5] [Show and tell]
Dans l'idée de créer une communauté *up-to-date*, les séances prévues offrent l'occasion de faire parler tout le monde! Tout élément sous le thème *e-ID et l'infrastructure de confiance numérique* pourrais être présenté: projets, produits, initiatives ou organisations actives.

**Aurais-tu envie de presenter qqch?** Réponds ici avec une bref explication, le sujet principal de ton *speech* et la durée envisagée de plutôt 5, 10 ou 15 minutes.

**Puis la communauté décide avec des *Up-Votes*,** quelles sujéts sont les plus pertinentes.

-----

Ganz in der Idee, eine Community zu schaffen, welche up-to-date ist, bieten die kommenden Online-Meetings die Möglichkeit, jedem eine Plattform für Präsentationen zu bieten. Im Zeichen der Themen *E-ID und digitale Vertrauensinfrastruktur* können Projekte, Produkte, Initiativen sowie Organisationen vorgestellt werden.

**Möchtest du einen Präsentations-Slot?** Dann erkläre in der Antwort kurz, was der Fokus wäre und ob du dafür eher 5, 10 oder 15 Minuten brauchst.

**Die Community wünscht anschliessend durch *Up-Votes*,** was denn möglichst bald dran genommen werden sollte.

-----

In linea con l'idea di creare una comunità aggiornata, i prossimi incontri online offrono l'opportunità di fornire a tutti una piattaforma per le presentazioni. Sotto i temi *E-ID e infrastruttura di fiducia digitale*, presentazioni di progetti, prodotti, iniziative e organizzazioni sono possibile.

**Se desiderate uno spazio per la presentazione,** spiegate brevemente nella vostra risposta quale sarebbe l'obiettivo e se avete bisogno di 5, 10 o 15 minuti.

**La comunità desidera allora, attraverso i *voti*,** quello che dovrebbe essere ripreso il più presto possibile.

**Comment by ursfis** [comment]
Health Info Net AG (HIN) betreibt eine SSI Infrastruktur mit der Möglichkeit einfach Usecases im Gesundheitswesen abzubilden und zu testen. HIN entwickelt die Plattform zusammen mit Switch (Ch. Graf) und More Than Bits (This Loepfe) laufend weiter. Mit der Plattform werden folgende Ziele verfolgt:
- Sammeln von Erfahrungen für einen späteren produktiven Einsatz
- Testen was heute schon möglich ist 
- Schaffung Akzeptanz bei den Nutzern und Stakeholdern
- Aufbau von Ökosystemen und Verständnis der verschiedenen Rollen
- exemplarische Umsetzung von organisations- und sektorübergreifenden Anwendungsfällen
- Sammeln von Anforderungen an die E-ID und das Governance Framework

Mit einer Präsentation von ca. 15Min. möchten wir unsere Infrastruktur, die Ausbaumöglichkeiten sowie die Möglichkeiten zur Partizipation vorstellen.
Titel der Präsentation:
"Verifiable Credentials für Gesundheitsfachpersonen"

**Reply by christianheimann** [reply]
Wie sehr schaut ihr in diesem Porjekt nach Grossbritannien betreffend [UK Digital Staff Passport](https://beta.staffpassports.nhs.uk/), welches quasi den hürdenlosen Wechsel von Spezialisten und Fachpersonen erlauben sollte?

**Reply by ursfis** [reply]
Ist ein sehr spannender Ansatz, den wir in unsere Überlegungen einbeziehen

**Reply by ChristophGraf-SWITCH** [reply]
In der Tat spannend. Habe bei den Autoren nachgefragt, wie die VCs für Skills aufgebaut sind. Waiting for reply...

**Comment by syh-42** [comment]
SSI Initiative 
(Version française ci-dessous) 
Die SSI-Initiative baut ein SSI-Ökosystem auf, um praktische Erfahrung im Bereich SSI zu sammeln und die folgenden Fragen zu beantworten:

- Technologie: Was ist die Maturität und die Komplexität von SSI in Bezug auf die Entwicklung und den Betrieb? Wie sieht der passende Technologie-Stack für die jeweiligen Anwendungen aus (Open Source vs. Produkt) 
- UX: Wie soll die User Experience gestaltet werden, um eine hohe Nutzerakzeptanz sicherzustellen und das Konzept der Endnutzer-Wallet verständlich zu machen? 
- Business: Was ist der konkrete Mehrwert für die Ökosystem-Teilnehmer
- Compliance: Welche Governance-Grundsätze, Prozesse und best Practices sind notwendig, um eine optimale Compliance zu gewährleisten

Das SSI-Ökosystem besteht momentan aus drei aufeinander aufbauenden Use Cases, die die drei Ambitions-Niveaus widerspiegeln:

1. Ausstellen einer Basis-ID auf der Grundlage einer SwissID
2. Ausstellen einer Wohnsitzbescheinigung im eGov-Portal des Kanton Aargaus.
3. Initiieren die vorläufigen Verkehrsberechtigung eines Fahrzeugs im cardossier-Portal

Use Case 1 und 2 sind seit September 2021 live. Use Case 3 wird derzeit umgesetzt.
Teilnehmer der Initiative sind: SwissSign, Kanton Aargau, der Verein cardossier und Adnovum. 
Gerne können wir in einer 15-minutigen Präsentation unser Vorgehen, den aktueller Stand (inkl. Demo) und die bis jetzt gewonnene Erkenntnisse präsentieren.

---------

L'initiative SSI met en place un écosystème afin d'acquérir une expérience pratique dans ce domaine et de répondre aux questions suivantes:

- Technologie : quelle est la maturité et la complexité de la SSI en termes de développement et d'exploitation ? Quelle est la solution appropriée pour chaque application (open source vs produit) ? 
- UX : Comment l'expérience utilisateur doit-elle être conçue pour garantir une adaptation élevée et rendre le concept de Wallet compréhensible ? 
- Business : quelle est la valeur ajoutée concrète pour les participants à l'écosystème ?
- Compliance : quels sont les principes de gouvernance, les processus et les meilleures pratiques nécessaires pour garantir une conformité optimale ?

L'écosystème SSI se compose actuellement de trois cas d'utilisation successifs, qui reflètent les trois niveaux d'ambition définit dans le document de travail e-ID:

1. émettre une  ID de base sur la base d'une SwissID
2. émettre une attestation de domicile sur le portail eGov du canton d'Argovie.
3. initier l'autorisation provisoire de circuler d'un véhicule sur le portail cardossier.

Les cas d'utilisation 1 et 2 sont en ligne depuis septembre 2021. Le cas d'utilisation 3 est en cours de réalisation. 
Les participants à l'initiative sont : SwissSign, le canton d'Argovie, l'association cardossier et Adnovum. 
Nous pouvons volontiers présenter notre démarche, l'état actuel (démo inclusive) et les enseignements acquis jusqu'à présent dans le cadre d'une présentation de 15 minutes.

**Reply by christianheimann** [reply]
Punkto Wohnsitzbestätigung, seid ihr da im Austausch mit den SBB, welche den gleichen digitalen Nachweis anstreben?

**Reply by ChristophGraf-SWITCH** [reply]
Die digitale Wohnsitzbestätigung scheint recht stark nachgefragt zu sein. Wir sehen auch im Bildungsbereich Anwendungen hierfür beim Onboarding zur Erstausbildung im Tertiärbereich.

**Reply by StephaneMingot** [reply]
Ja, wir sind im Austausch mit den SBB u.a. im Rahmen von DIDAS. Bei der SSI-Initiative ist es sogar erwünscht die herausgegebenen VC für andere Use Cases zu nutzen. Somit können weitere Use Case des Ambitionsniveaus 3 umgesetzt werden - und wir können u.a. gemeinsam lernen, welche Aspekte für Interoperabilität in einem grösseren Ökosystem relevant sind.  Bei der Umsetzung der weiteren Use Cases kann man auf die Erfahrungen und das Toolkit aus den Use Cases 1, 2 und 3 (mit Produkt- oder Open-Source-Lösungen) zurückgreifen.

**Reply by ursfis** [reply]
Die gegenseitige Nutzung von VC sollte das Ziel sein und würde wertvolle Erfahrungen für den Aufbau der Ökosysteme liefern

**Reply by Vitus-SBB** [reply]
Der Entscheid, ob das SBB-Projekt starten kann, wird nächste Woche fallen. Wir haben bereits festgestellt, dass es sich nicht um exakt dieselbe Wohnsitzbestätigung handelt. Ziel der Zusammenarbeit innerhalb von DIDAS (und hoffentlich auch bald innerhalb des "E-ID-Ökosystem-Sandkastens") ist es aber gerade, ein VC-Schema zu entwickeln, das all den oben erwähnten unterschiedlichen Bedürfnissen Rechnung tragen und somit standardisiert werden kann.

**Comment by svenstucki** [comment]
Ausführungen zur Inkompatibilität ISO digitaler Führerschein nach ISO 18013-5 <> SSI digitaler Führerausweis als Credential
  
Im September 2021 wurde der ISO 18013-5 Standard ("Mobile driving licence (mDL) application") verabschiedet. Dieser standardisiert Verfahren zur Signierung, Kontrolle und Austausch digitaler Führerscheine, welche zum Beispiel per Smartphone verwaltet werden können. Der Standard wurde unter anderem von grossen Technologieunternehmen wie Google und Apple mitentwickelt und wird von diesen aktiv unterstützt und besonders in den USA bereits eingesetzt. So unterstützt die Apple Wallet solche Führerscheine und Google bietet spezifische APIs in Android sowie Referenzimplementierungen an.
 
Der ISO 18013-5 Standard ist PKI basiert und wird gerne als "privacy-preserving" beschrieben. Gewisse Vorgaben machen es aber unmöglich, einen Standard-konformen mobilen Führerausweis zu implementieren und Privatsphäre auf dem Niveau einer modernen SSI zu erreichen. Wir zeigen die Herausforderungen auf, welche bei einer möglichen Kombination vom ISO mDL Standard mit SSI entstehen und wie diese kurz- und mittelfristig adressiert werden könnten.

**Comment by christianheimann** [comment]
@svenstucki , @ursfis und @syh-42 , wir würden gerne alle drei Themen im nächsten Online-Meeting aufnehmen. Ich kontaktiere euch bilateral, um Details dazu zu klären.
Alle andern dürfen sich auf spannende Inputs aus der Praxis freuen, am nächsten Online-Meeting am 3. März 2022!

**Reply by christianheimann** [reply]
Am Online-Meeting vom 3. März finden die Präsentationen von HIN und procivis statt. Das Input-Referat von @syh-42 wird auf Grund seiner Verfügbarkeit für das darauffolgende Online-Meeting vorgesehen.

**Comment by Jan-Friedli-digtalswitzerland** [comment]
[digitalswitzerland](https://digitalswitzerland.com/) hat im Januar eine Arbeitsgruppe lanciert zum Thema Ecosystem of Digital Credentials (EDC) in der Schweiz. Experten von +10 Organisationen arbeiten an einen Diskussionsbeitrag, welcher am 28. April veröffentlicht wird. Im “Spirit of Collaboration” arbeiten diese Experten zusammen, um vertrauenswürdige Informationen zu produzieren - aus dem Ökosystem für das Ökosystem!

Gerne würden wir diese Arbeitsgruppe und ihre Ziele kurz vorstellen. 10 Minuten sollten reichen. Besten Dank!

Link für mehr Informationen:
[https://landingpage.digitalswitzerland.com/expert-studio-ecosystem-of-digital-credentials](url)

**Comment by dlamers** [comment]
Hi all, I'm co-founder of Datakeeper, a digital identity wallet which was built taking the SSI design principles in mind. Datakeeper was founded within Rabobank, one of the largest banks in the Netherlands. Last 4 years we have created an UX friendly wallet and excellent secure and scalable bank-grade infrastructure to issue and verify credentials. Datakeeper is being used in the Netherlands to rent cars, commercial housing and social housing. Besides that we focus on IAM and consumer lending use cases. We disclose information from passport, bank accounts (through PSD2) and government sources in order to support those use cases. We are keen to inspire with our solution, share our experience and are looking for new partnerships and cooperations. If you're interested we would like to present in a time slot of 15 minutes. Thanks

**Comment by schischuMM** [comment]
**E-ID für die Schweiz - Der nächste Versuch mit Fokus SSI**

Im Rahmen unserer CAS Blockchain Transferarbeit haben wir uns mit Self Sovereign Identity (SSI) im Kontext der Schweizer E-ID auseinandergesetzt. 

Ziel dieser Transferarbeit war es herauszufinden, ob eine Blockchain-basierende Lösung einen zusätzlichen Nutzen gegenüber einer SSI-Lösung ohne Blockchain bringt und ob eine permissionless oder permissioned Blockchain sinnvoller wäre. Weiter sollten geeignete, Blockchain-basierte SSI E-ID Lösung im Rahmen eines Grobkonzeptes evaluiert werden. Es wurden die wichtigsten Use-Cases untersucht und daraus generische Use Cases definiert, gegen welche die Evaluation von geeigneten Lösungen erarbeitet wurden. Der Lösungsraum wurde auf Blockchain-basierte SSI-Anwendungen eingeschränkt.

Wir freuen uns die spannenden Erkenntnisse, welche wir aus unserer wissenschaftlichen Arbeit gewonnen haben, mit euch zu teilen.

Unsere Transferarbeit findet ihr unter diesem [LINK](https://compact.consulting/helvetia/CAS_BLC_2022.pdf) 

![helvetia_git](https://user-images.githubusercontent.com/2655674/164308968-cf835268-f4ff-4b6b-afbc-0a58214c99b4.png)

**Reply by pascalgottret** [reply]
Gerne dazu noch zwei Anmerkungen aus dem heutigen Online-Partizipationsmeeting:

- Permissionless DLT ist m.E. absolut der richtige Weg aus den im Paper genannten Gründen. Die Frage nach Interoperabilität stellt sich aber sowieso, unabhängig ob Permissionless oder nicht. Wir können nicht erwarten, dass in Zukunft alle Länder und alle Unternehmen auf die gleiche VDR zurückgreifen. Interoperabilität ist zwingend! Meines Erachtens aber gar kein Problem, denn alles was der Verifier braucht sind die richtigen DID Resolver der verschiedenen DID Methods anzusprechen. Da gibt es von der DIF bereits Lösungen dazu, die in Wallets integriert werden können (Universal Resolver). Problem ist damit vereinfacht gesagt gelöst.

- Die Erklärungen von Ingo Rübe soweit nachvollziehbar und richtig. Allerdings ist mir nicht ganz klar, wieso DIDs von Holders auf VDR gespeichert werden müssen. Solange ein Holder keine Credentials ausgibt, kann die DID dezentral auf seinem Wallet gespeichert werden ohne dass ein DID Document auf der VDR nötig ist (zumindest mit ZKP und BBS+). Gerne um genauere Erklärungen dazu.

**Reply by grt-dti** [reply]
wo war die Rede von Holder-DIDs in der VDR? Ich kann mir nur einen Anwendungsfall vorstellen, wo Verifier-DIDs im VDR stehen müssten (MiTM). Aber vielleicht braucht es gegen den MiTM der VC-Ausgabe auch die DID's der Verifier.....

**Reply by pascalgottret** [reply]
Ingo Rübe hat bei der Erklärung der Identifiers erklärt, dass die Daten (DID Documents) auf dem Registry gespeichert werden. So habe ich es zumindest verstanden. 
Als Holder habe ich eigentlich immer das Interesse zu prüfen (bzw. durch das Wallet prüfen zu lassen), wer der Verifier ist. Welcher Anwendungsfall meinst du? Bzw. bei welchem Anwendungsfall macht es keinen Sinn, den Verifier zu prüfen?

**Reply by grt-dti** [reply]
> Ingo Rübe hat bei der Erklärung der Identifiers erklärt, dass die Daten (DID Documents) auf dem Registry gespeichert werden. So habe ich es zumindest verstanden. Als Holder habe ich eigentlich immer das Interesse zu prüfen (bzw. durch das Wallet prüfen zu lassen), wer der Verifier ist. Welcher Anwendungsfall meinst du? Bzw. bei welchem Anwendungsfall macht es keinen Sinn, den Verifier zu prüfen?

Korrekt; das ist genau meine Aussage - für MiTM brauchen wir (verlässlich) die DIDs der Verifier, um "den korrekten Verifier" gegen den MiTM-Verifier unterscheiden zu können. Die Frage ist aber: anhand welcher Informationen erkennen wir die "Korrektheit"? Irgendwer muss diese "Korrektheit" der Zuordnung auf einem zweiten Kanal (z.B. den der VDR) verlässlich bestätigen.
Ich stellte aber die Frage, ob man diese MiTM-Mitigation (durch verlässliche Bestätigung des DID) nicht auch für den ganzen Issueing-Prozess / -vorgang brauchen. Liefert dies der Issueing-Prozess / -Protokoll bereits oder müsste man da noch etwas zusätzlich einbauen?

**Comment by AB-SB** [comment]
The Digital Identity draft report of the European Parliament ist out.
Have a look on the version of 31.05.2022:
https://www.europarl.europa.eu/doceo/document/ITRE-PR-732707_EN.pdf

**Comment by BorntraegerMarc** [comment]
@christianheimann Werden die Community-Slots in den Partizipations-Meetings noch durchgeführt?

Bei Interesse können wir (Amazon Web Services) in einem 15-20min Slot über unsere Erfahrungen mit globalen e-ID Projekten berichten. Hier ein paar zusätzliche Infos:

> Meet with AWS Blockchain Specialists to discuss AWS’ experience working with customers on self-sovereign identity / digital identity use cases, as well as AWS service offerings that can support such solutions.
> Agenda:
> 1. Introductions: AWS Blockchain Specialists
> 1. Customer success stories: Blockchain-based Digital Identity
> 1. AWS services that support digital identity workloads

**Reply by christianheimann** [reply]
Ja, Community-Beiträge werden weiterhin gemacht. Ich danke für den Input, wir werden uns für die Details direkt mit dir in Kontakt setzen.

**Comment by NSleumer** [comment]
As part of our Evening Talk Series, leading up to the SwissInformatics Congress on May 11th in Bern, Indukumar Vellapillil-Hari presented Aadhar, the E-ID system implemented in India. It was very well received and led to some interesting discussions. He would be willing to present it in this forum on March 2nd as well.
Regards,
Dr. Nora Sleumer
Vice President SwissInformatics

**Comment by NSleumer** [comment]
Hallo Christian,

Oh super. Indu ist Enterprise Architect bei der CS und hält sich den Slot frei am Donnerstag.
Ich kann dir seine Slides schicken, let me know. Er wird auf Englisch präsentieren.

In unsere SI E-ID Working Group mit viele Enterprise Architekten unterhalten wir uns natürlich über die Architektur. Das wäre vielleicht auch ein gutes Thema in Euer Forum. Security, Benutzung, Performance ... unser tägliche Arbeit.

Gruss,
Nora
________________________________
